src/components/inputs/Selector.tsx

```diff
@@ -65,7 +66,9 @@ export default function Selector({ getSelectedValue, startIcon, value, selectorT
               }
             }}
             getOptionLabel={(option: unknown) =>
-              selectorType === SelectorType.ASSIGNEE_SELECTOR ? ((option as IAssignee).name as string) : (option as string)
+              selectorType === SelectorType.ASSIGNEE_SELECTOR
+                ? ((option as IAssignee).name as string)
+                : ((option as WorkflowStateResponse).name as string)
```

Comment:
I get what this is doing but maybe we should move this to a separate function so it's cleaner. It will also help us with reusability since we will be needing avatars in many places

Reply:
Added in 6c24dc0675716b4337b4727af116339461a33886 commit